accounts/usbwallet/hub.go

```diff
@@ -185,8 +181,11 @@ func (hub *Hub) refreshWallets() {
 
 	for _, info := range infos {
 		for _, id := range hub.productIDs {
+			// We check both the raw ProductID (legacy) and just the upper byte, as Ledger
+			// uses `MMII`, encoding a model (MM) and an interface bitfield (II)
+			mmOnly := info.ProductID & 0xff00
 			// Windows and Macos use UsageID matching, Linux uses Interface matching
-			if info.ProductID == id && (info.UsagePage == hub.usageID || info.Interface == hub.endpointID) {
+			if (info.ProductID == id || mmOnly == id) && (info.UsagePage == hub.usageID || info.Interface == hub.endpointID) {
```

Comment:
This does not look right to me. Consider a future / non-compatible device, which identifies itself as `0x00aa`. 
`mmOnly := 0x00aa & 0xff00 // 0x0000`. The check `mmOnly == id` would match it as a `Ledger Blue`.

Reply:
I can't comment on if Ledger would release a product with an ID like this, but my understanding is that the 0x00** product IDs are legacy and new products would always follow the new MMII format.

An alternative might be to remove this product ID filtering altogether and just rely on the vendor ID lookup we're already doing.

Reply:
Removing the product id check would break support for legacy devices.

If they change tne scheme (which they might, it's only allowing for 256 devices which they might get to eventually) they will have to store them in the table one by one, or deal with it when the next scheme comes into effect. Maybe they'll change the vendor id as well. No way to know.